src/main/java/org/alliancegenome/curation_api/model/entities/AffectedGenomicModel.java

```diff
@@ -39,8 +34,6 @@
 @AGRCurationSchemaVersion(min = "1.5.0", max = LinkMLSchemaConstants.LATEST_RELEASE, dependencies = { GenomicEntity.class }, partial = true)
 public class AffectedGenomicModel extends GenomicEntity {
 
-	@FullTextField(analyzer = "autocompleteAnalyzer", searchAnalyzer = "autocompleteSearchAnalyzer")
-	@KeywordField(name = "name_keyword", aggregable = Aggregable.YES, sortable = Sortable.YES, searchable = Searchable.YES, normalizer = "sortNormalizer")
 	@Column(columnDefinition = "TEXT")
```

Comment:
Are we removing the AGM.name field from the index?

Reply:
It's replaced by the name field in the BiologicalEntity type bridge - you cannot have two indexes with the same name

Reply:
Ahh ok